rules/do-you-create-a-migration-plan/rule.md

```diff
@@ -23,14 +23,16 @@ Migrating from .NET Framework (4.x) to the latest .NET (5+) brings huge advantag
 ## Preparation
 The migration to a newer version of .NET is the perfect opportunity for you to take stock of your current application architecture, and address any technical debt your app has accumulated. Trying to migrate an application that's poorly architected or carrying a lot of tech debt will make your migration **exponentially harder**. Therefore, now is the time to perform a full audit of your app and ensure you have PBIs to rectify these problems before you consider it "migration-ready".
 
-### Dependency analysis
+### Manual ependency analysis
 Imagine a typical N-tiered application. Over the course of its life, the lines between each tier will often get blurred, either accidentally or deliberately. This can result in certain dependencies appearing where they shouldn't - such as `System.Web` references showing up in your application or data layer. This is a very common code smell and a great starting point to cleaning up your app.
 
 If your app has 3rd party dependencies (e.g. with a financial system, reporting system, etc.) - now is the time to investigate those integration points to determine whether those services provide compatible libraries and how those libraries differ (if at all). Create PBIs for these as well. 
 
 ### Infrastructure
 If you host your app on premise, it's also worth checking your infrastructure to ensure it has the necessary runtimes.
 
+![Figure: Install necessary .NET 8 runtimes](dotnet-runtimes.png)
```

Comment:
What are the arrows pointing at?

I think what we need here is to have an arrow pointing at a specific runtime and in the figure explain that we can see the correct runtime is installed to match our application e.g. 8.0.0

Reply:
I am not sure I am understanding this. The arrows are pointing to the all 8.0 runtimes as I have install all for Desktop, ASP.NET Core and .NET Core. Could you please be more specific?

Reply:
@YazhiChen as discussed, it seems that we only need ASP.NET Core and .NET runtime to run an application.
Please add a side-by-side picture as Piers requested to show TargetFramework and the required installed runtimes.

Also I'd block out the unnecessary runtimes with a box to prevent reader from being overwhelmed 😄

Reply:
Done.